cmake/developer_package/packaging.cmake

```diff
@@ -20,9 +20,17 @@ function(ie_cpack_set_library_dir)
         set(IE_CPACK_RUNTIME_PATH runtime/lib/${ARCH_FOLDER}/$<CONFIG> PARENT_SCOPE)
         set(IE_CPACK_ARCHIVE_PATH runtime/lib/${ARCH_FOLDER}/$<CONFIG> PARENT_SCOPE)
     else()
-        set(IE_CPACK_LIBRARY_PATH runtime/lib/${ARCH_FOLDER} PARENT_SCOPE)
-        set(IE_CPACK_RUNTIME_PATH runtime/lib/${ARCH_FOLDER} PARENT_SCOPE)
-        set(IE_CPACK_ARCHIVE_PATH runtime/lib/${ARCH_FOLDER} PARENT_SCOPE)
+        if (NOT LINUX_OS_NAME STREQUAL "CHROMIUMOS")
+            set(IE_CPACK_LIBRARY_PATH runtime/lib/${ARCH_FOLDER} PARENT_SCOPE)
+            set(IE_CPACK_RUNTIME_PATH runtime/lib/${ARCH_FOLDER} PARENT_SCOPE)
+            set(IE_CPACK_ARCHIVE_PATH runtime/lib/${ARCH_FOLDER} PARENT_SCOPE)
+            set(IE_CPACK_INCLUDES_PATH runtime/include PARENT_SCOPE)
+        else()
+            set(IE_CPACK_LIBRARY_PATH ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR} PARENT_SCOPE)
```

Comment:
- do we need explicit `CMAKE_INSTALL_PREFIX` here?
- we need to include https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html to have `CMAKE_INSTALL_LIBDIR` available 
- can we also use `CMAKE_INSTALL_INCLUDEDIR`?

Reply:
Updated the code with the changes